src/components/Team.tsx

```diff
@@ -13,26 +15,28 @@ const Team = () => {
 
   const [state, setState] = useState<TeamViewModel | undefined>();
 
+  const init = false;
+  const { team } = useParams();
   useEffect(() => {
-    const url = new URLSearchParams(window.location.search);
-    const teamNameString = url.get('teamName');
-    logger.debug(`Retrieved teamName ${teamNameString}`);
-    if (teamNameString) {
-      
-      const fetchTeamMembers = async () => {
-        return await DbProvider.getTeam(teamNameString);
-      };
-      fetchTeamMembers().then(data => {
-        setState({teamName: teamNameString, members: data.members});
-      });
+    if (!init) {
```

Comment:
Do we need `init` if it's never set to `true`? 

I think we should also consider adding a new task for the  "sad" path. What if `team` is not found? Should we render some kind of an error message?

Reply:
I might've missed it it should be assigned, I'll send an update.  And yeah we should probably render an error